samples/zigbee/light_switch/README.rst

```diff
@@ -135,6 +135,17 @@ FEM support
 
 .. include:: /includes/sample_fem_support.txt
 
+Configurable transmission power
+===============================
+
+In order to achieve a lower power consumption of the light switch, its transmission power can be configured using the :kconfig:option:`CONFIG_LIGHT_SWITCH_CONFIGURE_TX_POWER` Kconfig option.
+Per-channel transmission power in dBm is selected with :kconfig:option:`CONFIG_LIGHT_SWITCH_TX_POWER`.
+This affects to all frames sent by the device, even in the network scan phase.
+
+When :kconfig:option:`CONFIG_ZIGBEE_CHANNEL_SELECTION_MODE_MULTI` is set to `y`, :kconfig:option:`CONFIG_ZIGBEE_APP_CB_QUEUE_LENGTH` must be increased depending on the channel mask.
+In case the 16 channels are active, a proper value for :kconfig:option:`CONFIG_ZIGBEE_APP_CB_QUEUE_LENGTH` would be `17`.
```

Comment:
```suggestion
For example, in case 16 channels are active, a proper value for :kconfig:option:`CONFIG_ZIGBEE_APP_CB_QUEUE_LENGTH` would be ``17``.
```